cfme/tests/containers/test_containers_default_services.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from utils import testgen
+from utils.appliance.implementations.ui import navigate_to
+from utils.version import current_version
+from cfme.containers.service import Service, list_tbl
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+
+DefaultServices = ['frontend',
+                   'frontend-prod',
+                   'jenkins',
+                   'kubernetes',
+                   'router']
+
+
+# CMP-9884
+
+def test_containers_default_services():
+    """This test ensures that default container services are existing after provider setup
+
+    Steps:
+        * Go to Compute -> Container services.
+
+    Expected result:
+        ALL DefaultServices above should exist (appear in the table).
+    """
+    navigate_to(Service, 'All')
+    names = [r[2].text for r in list_tbl.rows()]
+    for serv in DefaultServices:
+        assert serv in names
```

Comment:
Stylistic Suggestion:
Instead of looping here, you could simply make ```DefaultServices``` a Python Set. Then something like ```names={r[2].text for r in list_tbl.rows()}``` and then just assert that ```names == DefaultServices```